src/UI/Implementation/Component/Glyph/Glyph.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/* Copyright (c) 2016 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Glyph;
+
+use ILIAS\UI\Component as C;
+use ILIAS\UI\Component\Counter\Counter;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
+
+class Glyph implements C\Glyph\Glyph {
```

Comment:
I'm missing the possibility to set an id here. I know, we don't want to think of HTML by using UI elements, but an id should be abstract enough (this applies also for all other UI elements).